core/ecschema-metadata/src/test/Metadata/Class.test.ts

```diff
@@ -59,6 +59,34 @@ describe("ECClass", () => {
       expect(await entityClass.getInheritedProperty("PrimProp")).to.be.undefined;
     });
 
+    it("successfully executes multiples calls to getProperties", async() => {
+      const baseClass = new EntityClass(schema, "Parent");
```

Comment:
We discussed in our daily call that construction of a schema is better done from JSON instead of doing it like this. Please change this test even if there are others in the same file which do this.